pkg/virt-operator/resource/apply/core.go

```diff
@@ -463,145 +464,112 @@ func (r *Reconciler) createOrUpdateRbac() error {
 	return nil
 }
 
-func (r *Reconciler) createOrUpdateConfigMaps() error {
```

Comment:
I understand this isn't used at the moment, but how do we explain that there's a list of ConfigMaps on the install strategy stucture, but we only reconcile one that's related to our CA. 

Also, what's the history of config map usage at install time. If we recently managed more configmaps, then we'd need to leave this function around for compatiblity when updating from old KubeVirt versions

Reply:
As far as I can tell the configmap resource that is not a CA and the associated [sync code](https://github.com/kubevirt/kubevirt/pull/3133/files#diff-f5fa8e6ec51d9110d819dbb3eb3f3246cc221c958c0dfce503d269ac1516458dR661) was added in PR https://github.com/kubevirt/kubevirt/pull/3133 but that was then reverted in PR https://github.com/kubevirt/kubevirt/pull/3498. but the revert PR didn't include removing the sync loop.

Reply:
great find. Yep that looks like it. And since that was never released, it's safe to remove the code.